src/GridGenerator.js

```diff
@@ -9,6 +10,27 @@ class GridGenerator {
     return null;
   }
 
+  static ring (center, mapRadius) {
+    let hexas = [];
+    let hex = HexUtils.add(center, HexUtils.multiply(HexUtils.direction(4), mapRadius));
+    for (let i = 0; i < 6; i++) {
+      for (let j = 0; j < mapRadius; j++) {
+        hexas.push(hex)
+        hex = HexUtils.neighbour(hex, i)
+      }
+    }
+    return hexas
+  }
```

Comment:
One very small addition that I would make for consistency, to add semicolons to line endings :)
I know they're optional and everything, but would nice!

Reply:
Sure. No problem. Sorry about that.